### ISSUE TITLE
Specify superclass of a class def

### ISSUE THREAD

**Issue by yhara** [yhara/shiika#70]
eg.

```
class A; end
class B : A; end
```

**Comment by yhara** [comment]
The symbol is `:`, not `<` like Ruby. This is because Shiika has parametric types

```
class B<T> < A

class B<T> : A
```

**Comment by yhara** [comment]
B should inherit methods, ivars and constants from A (maybe I'll split into each issue.)

**Comment by yhara** [comment]
Closed by #74. Created #72 and #73
